src/views/Receiving.vue

```diff
@@ -65,6 +68,9 @@ export default defineComponent({
       }
       await this.store.dispatch("shipment/findShipment", payload);
     },
+    loadMoreShipments(){
+      this.getShipments(process.env.VUE_APP_VIEW_SIZE , this.shipments.length / process.env.VUE_APP_VIEW_SIZE);
```

Comment:
Please consider the case when view size is 10 and shipment list has 25 records. I think this may break for all similar cases. Please fix this code

Reply:
Added `Math.ceil` to properly calculate the viewIndex